### ISSUE TITLE
Error: AttributeError: 'Part' object has no attribute 'image'

### ISSUE THREAD

**Issue by dkgrieshammer** [ssine/pptx2md#43]
Thanks a lot for the script. While testing the conversion, I get the following error. Not sure what to do, do I need to adjust a image in the presentation? How is pptx2md handling graphical forms like box / ellipse etc. drawn in ppt, btw?
  
```
File "/opt/homebrew/Caskroom/miniconda/base/lib/python3.10/site-packages/pptx/parts/slide.py", line 30, in get_image
    return self.related_part(rId).image
AttributeError: 'Part' object has no attribute 'image'
```

**Comment by ssine** [comment]
would you please attach a minimal pptx file that reproduces this error?

**Comment by drcicero** [comment]
I think I encountered the same problem, and temporarily added the following lines to parser.py, to ignore the issue as a workaround, and print a message for every missing picture:

    215       elif shape.shape_type == MSO_SHAPE_TYPE.PICTURE:
    216         try:
    217           notes += process_picture(shape, idx + 1)
    218         except AttributeError as e:
    219           print(e)

**Comment by ssine** [comment]
this catch logic is now published in version 1.6.0.
